WordPress/gutenberg
# Global styles: move preload paths filter to 6.7 (previously 6.8)

## What?

Follow up to:

- https://github.com/WordPress/gutenberg/pull/66468

https://github.com/WordPress/gutenberg/pull/66468 will be [cherry picked to 6.7](https://github.com/WordPress/gutenberg/pull/66542) since it's a regression.



## Why?
Given that this fixes a 6.7 regression move the code into 6.7 compat.

## How?
Copy paste

## Testing Instructions

Fire up this branch and head to the site editor.

Check that global-styles endpoints aren't fetched client side, but are, rather, preloaded via [rest_preload_api_request](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api.php#L2918)

Do the same for the post editor.


## Screenshots or screencast
